staging/src/k8s.io/code-generator/cmd/import-boss/README.md

```diff
@@ -0,0 +1,63 @@
+## Purpose
+
+- `import-boss` enforces import restrictions in a given repository.
+
+## How does it work?
+
+- When a directory is verified, `import-boss` looks for a file called `.import-restrictions`. If this file is not found, this directory and it's sub-directories will be skipped.
```

Comment:
`it's` -> `its`

this is a type that came from my suggestion, sorry.
https://github.com/kubernetes/kubernetes/pull/85564#discussion_r392239151

Reply:
`this directory and it's sub-directories will be skipped.`
If sub-directories contains .import-restrictions file, they are checked with the file.
So we can say just `this directory will be skipped.`